plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/dataexport/CommonElementExporter.java

```diff
@@ -464,8 +465,9 @@ private void addParam(final IInterfaceElement ie) throws XMLStreamException {
 		final boolean hasAttributes = hasNonTrivialAttributes(ie);
 		final boolean hasInitalValue = (ie instanceof final VarDeclaration varDecl) && (varDecl.getValue() != null
 				&& varDecl.getValue().getValue() != null && !varDecl.getValue().getValue().isBlank());
-		final boolean hasComment = ie instanceof final VarDeclaration varDecl && varDecl.getComment() != null
-				&& !varDecl.getComment().equals(""); //$NON-NLS-1$
+		final boolean hasComment = (ie instanceof final VarDeclaration varDecl && varDecl.getComment() != null
```

Comment:
is it enough here for you to ask for IInterfaceElement and not duplicate the check? this also includes e.g. adapters but they also have comments so that should not be a problem?

Reply:
As all interface elements may have comments I find that a good idea.